packages/domain/lib/model/base_request.dart

```diff
@@ -0,0 +1,3 @@
+abstract class BaseRequest {
+  Map<String, dynamic> toMap();
+}
```

Comment:
this class is not used anywhere by you and is definitely located in the folder for other purposes - remove it